contrib/Installer/boinc/boinc/modPersistedDataSystem.vb

```diff
@@ -1118,7 +1121,10 @@ TryAgain:
             Threading.ThreadPool.QueueUserWorkItem(AddressOf ThreadGetRac, oNeuralType)
 
 ThreadStarted:
-            mlQueue += 1
+            Dim mlLock As New Object
```

Comment:
Will this not create a new lock for each thread? We should be using a shared lock here, right?